wandb/integration/monai/stats_handler.py

```diff
@@ -0,0 +1,204 @@
+from typing import Any, Callable, Sequence, Union
+
+import torch
+from monai.utils import is_scalar
+
+import wandb
+from wandb.util import get_module
+
+DEFAULT_TAG = "Loss"
+ignite_engine = get_module("ignite.engine")
+
+
+class WandbStatsHandler:
+    def __init__(
+        self,
+        iteration_log: Union[
+            bool, Callable[[ignite_engine.Engine, int], bool], int
+        ] = True,
+        epoch_log: Union[bool, Callable[[ignite_engine.Engine, int], bool], int] = True,
+        epoch_event_logger: Union[
+            Callable[[ignite_engine.Engine, Any], Any], None
+        ] = None,
+        iteration_event_logger: Union[
+            Callable[[ignite_engine.Engine, Any], Any], None
+        ] = None,
+        output_transform: Callable = lambda x: x[0],
```

Comment:
I think this is an important argument and many a times the default setting might not be sufficient. I think this should show up earlier.